src/v3/src/components/ReminderPrompt/ReminderPrompt.tsx

```diff
@@ -27,6 +28,9 @@ export const DEFAULT_TIMEOUT_MS = 30_000;
 const ReminderPrompt: UISchemaElementComponent<{
   uischema: ReminderElement
 }> = ({ uischema }) => {
+  const {
+    loading = false,
```

Comment:
nit: I think setting it to default isn't necessary since loading is a required prop in the context.

Reply:
Yeah I was testing and meant to take that out lol oops.  Thank you!